app/models/intake.rb

```diff
@@ -6,7 +6,7 @@
 #  additional_info                                      :string
 #  adopted_child                                        :integer          default(0), not null
 #  already_applied_for_stimulus                         :integer          default(0), not null
-#  already_filed                                        :integer          default(0), not null
+#  already_filed                                        :integer          default("unfilled"), not null
```

Comment:
in a previous PR, "already filed" got promoted into this shared intake file, which means the enum is defined in this file. When that is the case, whatever does this logic looks to what the integer maps to as the string in the enum is used instead of 0. So for example, already_applied_for_stimulus is only defined in GyrIntake, so it says 0 even though 0 maps to unfilled.

Reply:
The reason it only applied in this PR is because the change in the previous PR must have been made after a db migration, and then these comments get autoupdated after this PR's migration.